src/singular.js

```diff
@@ -51,12 +59,13 @@ var Singular = {
       Singular.rpFrameLoaded = true;
     };
 
-    document.addEventListener('DOMContentLoaded', function () {
-      window.parent.Singular = Singular;
-
-      document.body.appendChild(opIframe);
-      document.body.appendChild(rpIframe);
-    });
+    if (Singular.properties.domContentLoaded) {
```

Comment:
Given no configuration changes, this should be backwards compatible. However, I'd prefer we guarded the property a bit more. With this check, if a end user adds a configuration property `'false'`, they will inadvertently append the frames on initialization instead of listening for a `DOMContentLoaded` event. If we want to protect previous functionality, we should restrict this other mode as much as possible.

This conditional check will be enabled with any property value that is not falsey (`false`, `0`, `undefined`, `null`, `''`)

Reply:
We will explicitly check whether the property has the value `true`.